db/main/migrate/20170713162000_add_last_cc_digists_to_invoices.rb

```diff
@@ -0,0 +1,7 @@
+class AddLastCcDigistsToInvoices < ActiveRecord::Migration
+  def change
+    change_table :invoices do |t|
+      t.string :last_cc_digists
```

Comment:
I think calling it: `cc_last_digits` would be better. We call it like that on Subscription.
There's also a typo on `digits`